charts/backstage/values.yaml

```diff
@@ -122,6 +122,7 @@ backstage:
   extraVolumeMounts: []
   extraVolumes: []
   initContainers: []
+  installDir: /app
```

Comment:
Probably worth having a description about what this field is, just so it's generated and added to the `README.md`. We should probably add some descriptions to the values above as well at some point in the future